test/jdk/java/awt/print/Dialog/DialogType.java

```diff
@@ -21,53 +21,66 @@
  * questions.
  */
 
-/**
+import java.awt.print.PrinterJob;
+import java.lang.reflect.InvocationTargetException;
+import javax.print.attribute.HashPrintRequestAttributeSet;
+import javax.print.attribute.PrintRequestAttributeSet;
+import javax.print.attribute.standard.DialogTypeSelection;
+import jtreg.SkippedException;
+
+/*
  * @test
  * @bug 6568874
+ * @key printer
+ * @library /java/awt/regtesthelpers
+ * @library /test/lib
+ * @build PassFailJFrame
+ * @build jtreg.SkippedException
  * @summary Verify the native dialog works with attribute sets.
- * @run main/manual=yesno DialogType
+ * @run main/manual DialogType
  */
 
-import java.awt.print.*;
-import javax.print.attribute.*;
-import javax.print.attribute.standard.*;
-
 public class DialogType {
 
-  static String[] instructions = {
-     "This test assumes and requires that you have a printer installed",
-     "It verifies that the dialogs behave properly when using new API",
-     "to optionally select a native dialog where one is present.",
-     "Two dialogs are shown in succession.",
-     "The test passes as long as no exceptions are thrown, *AND*",
-     "if running on Windows only, the first dialog is a native windows",
-     "control which differs in appearance from the second dialog",
-     ""
-  };
+    static String instruction = """
+            This test assumes and requires that you have a printer installed.
+            It verifies that the dialogs behave properly when using new API
+            to optionally select a native dialog where one is present.
+            Two dialogs are shown in succession.
+            The test passes as long as no exceptions are thrown, *AND*,
+            if running on Windows only, the first dialog is a native windows
+            control which differs in appearance from the second dialog.
```

Comment:
The above comment is out of date. It is from before we had macOS which also has a native dialog.
Please update it.